lib/cartopy/tests/mpl/test_patch.py

```diff
@@ -4,13 +4,15 @@
 # See LICENSE in the root of the repository for full licensing details.
 
 from matplotlib.path import Path
+import pytest
 import shapely.geometry as sgeom
 
 import cartopy.mpl.patch as cpatch
 
 
-class Test_path_to_geos:
-    def test_empty_polygon(self):
+@pytest.mark.parametrize('legacy', [False, True])
```

Comment:
Maybe be a bit more explicit in the variable name about what is legacy here.
```suggestion
@pytest.mark.parametrize('use_legacy_path_to_geos', [False, True])
```